.codeclimate.yml

```diff
@@ -23,6 +33,8 @@ engines:
         enabled: false
       NestingDepth:
         enabled: false
+      MergeableSelector:
```

Comment:
I don't often agree with the linter on this one. I feel like us humans can organize the scss better.